### PR TITLE
PEP-8 compliance for all files in module

### PR DESCRIPTION
**Pull request by shacker**

### CODE REVIEW

File: pyItunes/__init__.py [hunk 1]
```diff
@@ -1,4 +0,0 @@
-from pyItunes.XMLLibraryParser import XMLLibraryParser
```
Comment (shacker) [inline pyItunes/__init__.py]: I felt a bit nervous about removing these from init, but couldn't figure out their purpose, and my project ran just fine without them. Confirm?

Comment (liamks) [inline pyItunes/__init__.py]: I think XMLLibraryParser may no longer be used. plistlib replaced it?

**Comment by liamks** [comment]
I'm good with merging this if it works :) . Also feel free to delete the actual XMLLibraryParser if it's no longer used (The actual file).

**Comment by shacker** [comment]
Added one more tweak for PEP-8. Will remove XMLLibraryParser in a separate PR.

**Comment by liamks** [comment]
@shacker cool - can you fix the merge conflict?

**Comment by shacker** [comment]
Resolved. Sorry for the delay.

**Comment by liamks** [comment]
Thanks!

**Pull request merged**